web_advanced_search_wildcard/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# Copyright 2016 Therp BV <http://therp.nl>.
+# Copyright 2017 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Wildcard in advanced search",
+    "summary": "Webmodule to add wildcard operators in advanced search field",
+    "version": "11.0.1.0.0",
+    "category": "web",
+    "license": "AGPL-3",
+    "author": "initOS GmbH & Co. KG, Therp BV, "
+              "Odoo Community Association (OCA)",
+    "application": False,
```

Comment:
drop this flags, as they are defaults. 
Add `'website': 'https://github.com/OCA/web',`